gitversion/gen_cpp_header/template/template_version.h

```diff
@@ -0,0 +1,10 @@
+#ifndef VERSION_H_
```

Comment:
place this file in gitversion/templates/cpp/version.h

Reply:
ok, changed!